package.json

```diff
@@ -34,11 +34,13 @@
   "dependencies": {
     "@babel/core": "^7.4.5",
     "@babel/polyfill": "^7.4.4",
+    "eslint-plugin-react": "^7.13.0",
     "html-webpack-plugin": "^3.2.0",
     "path": "^0.12.7",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-redux": "^7.0.3",
+    "react-router-dom": "^5.0.0",
```

Comment:
Hello @Amoswachira ,
The package `eslint-plugin-react` is already defined in line 32 of the package.json file as a dev dependency. You could remove it from the dependencies section to avoid redundancy.

Reply:
@Ogutu-Brian  it is solved